.mergify.yml

```diff
@@ -35,11 +35,17 @@ pull_request_rules:
         strict: false
       dismiss_reviews: {}
       delete_head_branch: {}
+    actions:
```

Comment:
I believe the section already exists above?

Reply:
hmm, not sure what happened, will remove that